Azure/sonic-buildimage
# [Ingrasys] change hwmon kernel modules installation order

* [Ingrasys] change hwmon kernel modules installation order

- What I did

change hwmon kernel modules installation order on S8810-32Q/S8900-54XC/S8900-64XC/S9100-32X

- How I did it

change hwmon kernel modules installation order in i2c_utils.sh and update related fancontrol file

- How to verify it

Check the hwmon kernel module insertion order in /sys/class/hwmon/hwmonX.
w83795 should be in /sys/class/hwmon/hwmon1, and jc42 should be the last one.

- Description for the changelog

change hwmon kernel modules installation order on S8810-32Q/S8900-54XC/S8900-64XC/S9100-32X

- A picture of a cute animal (not mandatory but encouraged)
